### DISCUSSION TITLE
OnMount for servessless functions?

### DISCUSSION THREAD

**Discussion by BlitzJB** [vercel/vercel#8345] [Help]
i am deploying an app that only consists of serverless functions.
I understand serverless functions only run ondemand, but is there a way to run a peice of code right after the functions are deployed?

**Comment by swarnava** [comment]
Could you elaborate your usecase?

**Reply by BlitzJB** [reply]
I am building a discord bot application and it will receive command invocations as post requests from discord, now these post requests are received and handled by my serverless function in vercel. Things work fine on that regard. These commands need to be registered to discord by sending them a http post. now i'd like to do that whenever my serverless function is redeployed
